lib/filterx/expr-literal-container.c

```diff
@@ -126,6 +141,23 @@ _literal_container_eval(FilterXExpr *s)
         }
 
       FilterXObject *value = filterx_expr_eval(elem->value);
+      if (elem->nullv)
+        {
+          if (!value)
+            filterx_eval_dump_errors("FilterX: null coalesce assignment suppressing error");
+
+          gboolean value_is_null_or_error = !value || filterx_object_is_type(value, &FILTERX_TYPE_NAME(null))
+                                            || (filterx_object_is_type(value, &FILTERX_TYPE_NAME(message_value))
+                                                && filterx_message_value_get_type(value) == LM_VT_NULL);
```

Comment:
just one question: 

The same null/error coalescing logic is implemented in other places as well (like expr-assign), shouldn't we introduce a filterx_object_is_coalescable() (or a better name)